heart/test/ionicmodels/TestCvodeCells.hpp

```diff
@@ -344,14 +350,14 @@ class TestCvodeCells : public CxxTest::TestSuite
         // Need to load dynamicly as we don't have config files anymore
            OutputFileHandler handler("TestCvodeCells", true);
 
```

Comment:
Don't know whether these continuous tests that refer to the couple of CellML files in chaste source should be changing? There used to be just a handful of CellML files in the source so they got auto-converted on building (rather than all of them) and then those were used (and re-used in many test files) for all the continuous test pack.